### DISCUSSION TITLE
Where has the documentation gone?

### DISCUSSION THREAD

**Discussion by Ipifo** [go-rel/rel#373] [General]
Hi,

I'm fairly new to Golang. About a year ago, I was getting to know Go and found using go-rel awesome. I love the migration and testing parts of Rel. 

I've just come back to working with it, after a bussy period, but it seems the documentation from https://go-rel.github.io/reference/rel/ has disappeared.  Is this comming  back?  

Out of all the ORM's for Go I've found this one to be the best but it doesn't seem to be getting any traction in any of the reviews I see.  Is Rel going to stick around?

**Comment by Ipifo** [comment]
I've realized I can use the Golang Package documentation.  

https://pkg.go.dev/github.com/go-rel/rel#section-documentation

It just didn't seem as user friendly as the previous documentation at https://go-rel.github.io/reference/rel/

**Reply by Fs02** [reply]
oops, seems there's error on the document generator
let me take a look it later

**Reply by Fs02** [reply]
should be fixed now
